test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -56,6 +57,11 @@ describe('IexecAccessors', async () => {
             dataset: datasetAddress,
             workerpool: workerpoolAddress,
         };
+        callbackAddress = await new TestClient__factory()
```

Comment:
Here the TestClient smart contract will be deployed before each it test ?

Reply:
It's deployed only once, then it's a network reset before each test (fixture mechanism of Hardhat).